node-group-project/src/components/PetTypeContainer.js

```diff
@@ -3,28 +3,34 @@ import { Link } from "react-router-dom";
 
 const PetTypeContainer = (props) => {
   const [adoptablePets, setAdoptablePets] = useState([]);
-
+  console.log("Pet type container")
```

Comment:
please remove console.logs prior to submitting a PR. 

This applies to all instances in the file.